app/src/main/java/com/kamron/pogoiv/ScreenShotHelper.java

```diff
@@ -0,0 +1,47 @@
+package com.kamron.pogoiv;
+
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.FileObserver;
+import android.support.v4.content.LocalBroadcastManager;
+
+import java.io.File;
+
+/**
+ * Created by Sarav on 9/9/2016.
+ */
+public class ScreenShotHelper {
+
+    private static ScreenShotHelper instance = null;
+    private FileObserver screenShotScanner;
+
+    private ScreenShotHelper(final Context context, final String screenShotDir) {
+        screenShotScanner = new FileObserver(screenShotDir, FileObserver.CLOSE_NOWRITE | FileObserver.CLOSE_WRITE) {
+            @Override
+            public void onEvent(int event, String file) {
+                if (file != null) {
```

Comment:
I'm assuming you saw the later commit that just deletes that flag? We don't really need that flag in addition to the ones in Pokefly. This is part of the effort to reduce it to 1 flag. Something like "isBusyWithUser" that denotes that the IV related UI is active. When that happens, we shouldn't need to process any screens or screenshots.

But then, it's going to be really hard to trigger that anyway. Even if you take a screenshot, the pokemon scan code won't see any pokemon and bail.